common/Cargo.toml

```diff
@@ -5,12 +5,15 @@ edition = "2021"
 
 # client
 [dependencies.substrate-api-client]
-git = "https://github.com/scs/substrate-api-client"
-rev = "aa4b84f8775f972e5967aca3d96dd163619f65d5"
+git = "https://github.com/Cardinal-Cryptography/substrate-api-client.git"
+rev = "8b7346a11a70f46d05de543a6bed37f3fdf6ac8f"
```

Comment:
Unfortunately this SHA corresponds to a latest commit at branch called `single_threaded_sender.PR`, which (pardon me) definitely does not sound like a source of a stable version for 3rd-party library :P
The easiest way is to rename the branch or create a tag.

Reply:
How is renaming of a branch going to fix anything :D

Reply:
I will make a tag of from it, since pushing it to main doesn't look like a good idea.

Reply:
changed to tag